doc/UsersGuide/BuildingRunningTesting/SRW-SD.rst

```diff
@@ -84,21 +81,23 @@ Users will need to configure their experiment by setting parameters in the ``con
    
 Users will need to change the ``ACCOUNT`` variable in ``config.yaml`` to an account they have access to. They may also wish to adjust other experiment settings. For more information on each task and variable, see :numref:`Section %s <ConfigWorkflow>`. 
 
-On Level 1 systems, users can find :term:`ICs/LBCs` for the SRW-SD sample case in the usual :ref:`input data locations <Data>` under ``RAP/2019072200``. Users will need to add the following lines to ``task_get_extrn_*:`` in their ``config.yaml`` file, adjusting the file path to point to the correct data locations:
+On Level 1 systems, users can find :term:`ICs/LBCs` for the SRW-SD sample case in the usual :ref:`input data locations <Data>` under ``RAP/2019072200``. Users will need to switch the data directory for :term:`ICs/LBCs` if running on Orion or Hercules, on the following lines in ``config.yaml/task_get_extrn_*:``:
 
 .. code-block:: console
 
    task_get_extrn_ics:
-     USE_USER_STAGED_EXTRN_FILES: true
-     EXTRN_MDL_SOURCE_BASEDIR_ICS: /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/develop/input_model_data/RAP/${yyyymmddhh}
+     EXTRN_MDL_SOURCE_BASEDIR_ICS: /scratch2/NCEPDEV/naqfc/Chan-hoo.Jeon/aqm_sample_data/RAP_DATA_SD/${yyyymmddhh} # hera
+#  EXTRN_MDL_SOURCE_BASEDIR_ICS: /work/noaa/epic/SRW-AQM_DATA/input_model_data/RAP/${yyyymmddhh} # orion/hercules
    task_get_extrn_lbcs:
-     USE_USER_STAGED_EXTRN_FILES: true
-     EXTRN_MDL_SOURCE_BASEDIR_LBCS: /scratch1/NCEPDEV/nems/role.epic/UFS_SRW_data/develop/input_model_data/RAP/${yyyymmddhh}
+     EXTRN_MDL_SOURCE_BASEDIR_LBCS: /scratch2/NCEPDEV/naqfc/Chan-hoo.Jeon/aqm_sample_data/RAP_DATA_SD/${yyyymmddhh} # hera
+#  EXTRN_MDL_SOURCE_BASEDIR_LBCS: /work/noaa/epic/SRW-AQM_DATA/input_model_data/RAP/${yyyymmddhh} # orion/hercules
```

Comment:
Is there a reason not to put this data in the typical data locations? It's probably better not to use personal project space in the data paths, but I seem to recall discussing some reason why we were doing this... just don't remember what it was... ;)

Reply:
Sorry. I'll take a look at it again and get back to you soon :)

Reply:
@BenKoziol-NOAA @gspetro-NOAA, I've copied the files and updated the sample script.

Reply:
@chan-hoo - Thank you! I'll take a look at the script and update the documentation here.